File: Data Structures & Algorithms/two-integer-sum/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int n = nums.length; 
        HashMap<Integer, Integer> seen = new HashMap<>(); 
        int complement = 0; 
        for(int i = 0;i<n;i++) {
            complement = target - nums[i]; 
            if(seen.containsKey(complement))
            {
                return new int[] {seen.get(complement), i}; 
            }
            seen.put(nums[i],i); 
        }
        return new int[] {-1, -1}; 
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        res =[]
        hashmap = {}
        count = 0
        for i, a in enumerate(nums):
            if a in hashmap:
                if hashmap.get(a) < k:
                    res.append(a)
                    count += 1
            hashmap[a] = count
        return res
            